DEV GUIDE.md

```diff
@@ -0,0 +1,16 @@
+Install nvm to install multiples node envs
+Install node >=14.17.0 (i'm using 14.18.1)
+Install aragon ui or fix it in package json
+run
+    npm install --save @aragon/ui
+run 
+    yarn
+run 
+    yarn start
+
+Create an address on MetaMask.
+Use the Rinkeby TestNet
+
+Create an account on https://app.pinata.cloud/ and get the token on your env to REACT_APP_PINATA_API_TOKEN will convert to PINATA_API_TOKEN
+Select all property and get the JWT there
```

Comment:
Let´s state that this is only necessary to test the onboarding process.

Reply:
Perfect! Will change that.

Reply:
So on that DEV Guide i told to install @aragon-ui too. So maybe its better get it DEV GUIDE off that PR, what you think?

Reply:
I'll make some PR to that to speed up that integration.